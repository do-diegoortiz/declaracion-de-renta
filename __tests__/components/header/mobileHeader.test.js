import React from 'react'
import { mount } from 'enzyme'
import { create } from 'react-test-renderer'

import MobileHeader from '../../../components/header/mobileHeader/mobileHeader'

describe('<MobileHeader />', () => {
  
  test('It render without crashing', () => {
    const mobileHeader = mount(<MobileHeader />)
    expect(mobileHeader.length).toEqual(1)
  })

  test('Check the component UI', () => {
    const mobileHeader = create(<MobileHeader />)
    expect(mobileHeader.toJSON()).toMatchSnapshot()
  })
})
