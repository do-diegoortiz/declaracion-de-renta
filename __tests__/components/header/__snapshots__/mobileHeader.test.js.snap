// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MobileHeader /> Check the component UI 1`] = `
<div>
  <img
    alt="Logo"
    src={Object {}}
  />
  <svg
    viewBox="0 30 500 80"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M0.00,49.98 C148.42,98.97 346.49,9.17 521.44,40.75 L500.00,150.00 L0.00,150.00 Z"
      fill="#FFF"
      fillOpacity="1"
    />
  </svg>
</div>
`;
